pkgs/by-name/ne/nexusmods-app/package.nix

```diff
@@ -34,6 +35,8 @@ buildDotnetModule rec {
   projectFile = "src/NexusMods.App/NexusMods.App.csproj";
   testProjectFile = "NexusMods.App.sln";
 
+  buildInputs = [ avalonia ];
```

Comment:
Does this mean other deps listed in this package can be dropped, and instead be depended on transitively?

Reply:
Yeah, I've now removed `runtimeDeps`, and everything still seems to work (there's a GUI and it has text).